common/util/json.go

```diff
@@ -23,57 +23,18 @@ import (
 	"errors"
 	"fmt"
 	"reflect"
-	"strconv"
 )
 
-var bytesT = reflect.TypeOf(Bytes(nil))
-
 // Bytes marshals/unmarshals as a JSON string with 0x prefix.
 // The empty slice marshals as "0x".
 type Bytes []byte
```

Comment:
Hmm it looks like `Bytes` is not actually used anywhere except `common/types/hashes.go`, where its usage is also removed in this patch